sdk/servicebus/Microsoft.Azure.WebJobs.Extensions.ServiceBus/src/Config/ServiceBusExtensionConfigProvider.cs

```diff
@@ -89,13 +91,14 @@ public void Initialize(ExtensionConfigContext context)
             context.AddBindingRule<ServiceBusAttribute>().Bind(bindingProvider);
         }
 
-        internal static void LogExceptionReceivedEvent(ExceptionReceivedEventArgs e, ILoggerFactory loggerFactory)
+        internal static void LogExceptionReceivedEvent(ProcessErrorEventArgs e, ILoggerFactory loggerFactory)
         {
             try
             {
-                var ctxt = e.ExceptionReceivedContext;
+                var errorSource = e.ErrorSource;
                 var logger = loggerFactory?.CreateLogger(LogCategories.Executor);
-                string message = $"Message processing error (Action={ctxt.Action}, ClientId={ctxt.ClientId}, EntityPath={ctxt.EntityPath}, Endpoint={ctxt.Endpoint})";
+                //TODO new SDK does not expose client ID in event args or on clients
+                string message = $"Message processing error (Action={errorSource}, EntityPath={e.EntityPath}, Endpoint={e.FullyQualifiedNamespace})";
```

Comment:
We can consider exposing the Identifier on the Processor and the event args.

Reply:
Yes, it is possible if a customer overrides the MessagingProvider and provides their own processors.